rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ClearingRequestSpecTest.java

```diff
@@ -236,4 +232,29 @@ public void should_document_get_clearingrequests_by_state() throws Exception {
                         )));
     }
 
+    @Test
+    public void should_add_comment_to_clearing_request() throws Exception {
+        Map<String, Object> comment = new LinkedHashMap<>();
+        comment.put("text", "comment text 1");
+        comment.put("commentedBy", "test.user@sw60.org");
+        String accessToken = TestHelper.generateAuthHeader(testUserId, testUserPassword);
+
+        this.mockMvc.perform(
+                        post("/api/clearingrequest/" + clearingRequest.getId() + "/comments")
+                                .contentType(MediaTypes.HAL_JSON)
+                                .content(this.objectMapper.writeValueAsString(comment))
+                                .header("Authorization", accessToken))
+                .andExpect(status().isOk())
+                .andDo(this.documentationHandler.document(
+                        requestFields(
+                                fieldWithPath("text").description("The text of the comment"),
+                                fieldWithPath("commentedBy").description("The user who made the comment")
```

Comment:
The field should not be provided by the user, but calculated by backend to prevent proxy.